src/api/index.ts

```diff
@@ -148,6 +148,27 @@ export function getAccountModules(
   );
 }
 
+export function getAccountModule(
+  requestParameters: {
+    address: string;
+    moduleName: string;
+    ledgerVersion?: number;
+  },
+  nodeUrl: string,
+): Promise<Types.MoveModuleBytecode> {
+  const client = new AptosClient(nodeUrl);
```

Comment:
instead of creating an AptosClient here, lets pass in a client instead in param so that when we wanna query different account modules, it doesn't create clients everytime.

Reply:
agree with your suggestion but this is what all other methods in the same file does.
if we want to do this, prefer doing in a separate PR